Shared/ThresholdKeyView.swift

```diff
@@ -57,6 +57,7 @@ struct ThresholdKeyView: View {
                                 // try? await KeychainInterface.syncShare(threshold_key: threshold_key, share_index: nil)
```

Comment:
Where is the save and fetch logic from #4.

Reply:
I'll apply this on another PR